runtime/common/xwalk_content_client.cc

```diff
@@ -45,6 +51,69 @@ const char kPnaclPluginExtension[] = "";
 const char kPnaclPluginDescription[] = "Portable Native Client Executable";
 #endif
 
+#if defined(ENABLE_PLUGINS)
+const int32 kPepperFlashPermissions = ppapi::PERMISSION_DEV |
+                                      ppapi::PERMISSION_PRIVATE |
+                                      ppapi::PERMISSION_BYPASS_USER_GESTURE |
+                                      ppapi::PERMISSION_FLASH;
+
+content::PepperPluginInfo CreatePepperFlashInfo(const base::FilePath& path,
+                                                const std::string& version) {
+  content::PepperPluginInfo plugin;
+
+  plugin.is_out_of_process = true;
+  plugin.name = content::kFlashPluginName;
+  plugin.path = path;
+  plugin.permissions = kPepperFlashPermissions;
+
+  std::vector<std::string> flash_version_numbers;
+  base::SplitString(version, '.', &flash_version_numbers);
+  if (flash_version_numbers.size() < 1)
```

Comment:
These functions are just copied from Chromium, I will update the commit msg. Thanks.